requirements.txt

```diff
@@ -8,12 +8,16 @@ amazon-braket-sdk==1.59.1
 amazon-braket-algorithm-library==1.4.1
 cvxpy==1.4.1
 ipykernel==6.25.2
+jax==0.4.10
+jaxlib==0.4.10
 jinja2==3.1.2
 markupsafe==2.1.3
 matplotlib==3.8.0
 mypy-extensions==1.0.0
 openfermion==1.5.1
 openfermionpyscf==0.5
+optax==0.1.5
+oqpy==0.2.1
```

Comment:
This is pulled in from the BDK - do we need it declared?

Reply:
For `oqpy`? I am not sure. This example did not use `oqpy`. For `jax`, `jaxlib` and `optax`, we need these three for these particular versions.

Reply:
Yes, oqpy specifically. We pull this in via the BDK so this can be removed unless this is pinned for a specific reason